package.json

```diff
@@ -56,37 +53,55 @@
     "/bin",
     "/npm-shrinkwrap.json",
     "/oclif.manifest.json",
-    "/src"
+    "/lib"
   ],
   "homepage": "https://github.com/protofire/eth-cli#readme",
   "keywords": [
     "ethereum",
     "blockchain",
-    "cli"
+    "cli",
+    "oclif"
   ],
   "license": "MIT",
-  "main": "src/index.js",
+  "main": "lib/index.js",
   "oclif": {
-    "commands": "./src/commands",
+    "commands": "./lib/commands",
     "bin": "eth",
     "plugins": [
-      "@oclif/plugin-help",
-      "@oclif/plugin-not-found",
-      "@oclif/plugin-warn-if-update-available"
+      "@oclif/plugin-help"
     ]
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/protofire/eth-cli"
+  },
   "scripts": {
     "postpack": "rm -f oclif.manifest.json",
-    "posttest": "eslint src test",
-    "lint": "eslint src test",
-    "prepack": "oclif-dev manifest && oclif-dev readme",
+    "posttest": "jest ./test",
+    "prepack": "rm -rf lib && tsc -b && oclif-dev manifest && oclif-dev readme",
```

Comment:
Q: Thoughts about to add some new scripts right here, example:
```
    "lint": "tslint --project . 'src/**/*.{ts,tsx}' && npm run prettier-check",
    "prettier": "prettier --write '**/*.{ts,tsx,json,md}' --ignore-path .gitignore",
    "prettier-check": "prettier --check '**/*.{ts,tsx,json,md}' --ignore-path .gitignore"
```

Reply:
I'm OK with that.

Reply:
Done.